src/main/fc/fc_msp.c

```diff
@@ -1070,6 +1071,13 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         sbufWriteU8(dst, 0); // Transponder not supported
 #endif
         break;
+    case MSP_TRANSPONDER_PROVIDER:
```

Comment:
i think we can ditch the MSP_(SET_)TRANSPONDER_PROVIDER pair and just expand MSP_(SET_)TRANSPONDER_CONFIG pair.

perhaps the payload should be
byte 0: 
bits 0 = transponder supported 0 = no, 1 = yes
bits 3,2,1 = provider: 000 = none, 001 = ilap, 010 = arctimer
bits 7,6,5,4 = data-length in bytes

bytes 1 onwards - the transponder data. 

this way it would remain fairly compatible with existing UIs and would not need an additional pair of MSP commands